test/integration/run_int_test.go

```diff
@@ -281,6 +281,28 @@ func (suite *RunIntegrationTestSuite) TestRun_BadLanguage() {
 	cp.Expect("The language for this script is not supported", 5*time.Second)
 }
 
+func (suite *RunIntegrationTestSuite) TestRun_Perl_Variable() {
+	suite.OnlyRunForTags(tagsuite.Run)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.PrepareActiveStateYAML(strings.TrimSpace(`
+    project: https://platform.activestate.com/ActiveState-CLI/Perl-5.32?commitID=a4762408-def6-41e4-b709-4cb548765005
+	`))
+
+	cp := ts.SpawnWithOpts(
+		e2e.WithArgs("activate"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
```

Comment:
One other option here: `e2e.AppendEnv("PERL_VERSION=not_a_version")`; that way if this test fails, we'll get a very specific error from the `cp.Expect(...)` call later.

Reply:
Okay.